### PR TITLE
Ad/intern page

### PR DESCRIPTION
**Pull request by outstandy**
the ol' intern-a-rooski

👩‍🎓 Application criteria
📂 Dates + application form
📚 Learning resources

### CODE REVIEW

File: assets/css/_global.css [hunk 1]
```diff
@@ -178,6 +188,15 @@ a:not(.button) {
   }
 }
 
+.triangle {
+  /* background-image: url('../img/triangle-huge.png'); */
```
Comment (mordecai174) [inline assets/css/_global.css]: is this supposed to be commented out?

File: assets/css/_global.css [hunk 2]
```diff
@@ -178,6 +188,15 @@ a:not(.button) {
...
+  /* background-size: cover; */
```
Comment (jssee) [inline assets/css/_global.css]: can we drop this commented code?

File: assets/css/_carousel.css [hunk 3]
```diff
@@ -26,6 +26,10 @@
   }
 }
 
+.no-carousel {
```
Comment (jssee) [inline assets/css/_carousel.css]: I think a better solution would be to conditionally load the carousel script on the pages it is being used. This way we wont have useless JS blocking the load time on the internship page.

File: assets/css/_global.css [hunk 4]
```diff
@@ -74,7 +74,7 @@ a {
   text-decoration: none;
 }
 
-a:not(.button) {
+a:not(.button, .resource-block) {
```
Comment (mordecai174) [inline assets/css/_global.css:77]: I noticed there's another rule just for `.resource-block` what do you need to inherit from here?

**Review by mordecai174 — DISMISSED** [review]

File: views/includes/internships/_hiring.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
+<section id="hiring" class="step g-section-spacer" data-themeset="dark">
+  <div class="content header-spacer">
+      <h3 class="beta fade-in an-beta">Who should apply?</h3>
+      <p class="sub-text fade-in an-alpha lead">Every designer starts out differently. We don’t require a specific background or experience level, but there’re a few things you should be be confident in:</p>
+      <ul class="content-list sub-text">
+        <li class="fade-in an-alpha">Your graphic and visual design skills</li>
```
Comment (jssee) [inline views/includes/internships/_hiring.html:6]: I like that you're animating the `<li>` elements in individually here, but the responsibility section below doesnt do the same. Can we animate those in as well so the behavior is consistent?

**Review by jssee — CHANGES_REQUESTED** [review]
Nice work, just have a few comments. I know we are trying to get this out soonish so feel free to disregard some of these and we can get back to it later.

There are also some design tweaks we might want to consider:
1. Theres something weird happening where the header background and the `.content` / `.hero` are slightly different shades of black
2. The intro section where the wavy triangle is under the text is hard to read, we could probably drop the opacity on the image. could just be me?
3. Theres some areas that i think we can add some type of visual hierarchy, like to the resource blocks and the work section, again this doesnt have to be addressed immediately

File: assets/css/_global.css (hunk 4 above)
Comment (outstandy) [reply assets/css/_global.css:77]: @mordecai174 the resource blocks are all anchor links; the base link styles overwrite them.

File: views/includes/internships/_hiring.html (hunk 5 above)
Comment (mordecai174) [reply views/includes/internships/_hiring.html:6]: The timing on some of the sections seems off. I know it's probably just some growing pains from trying to to apply the index page styles to this page. We need to revisit these at some point. If there's anything quick and easy to do in the meantime, let's hit it. If not we can come back to it.

**Review by jssee — APPROVED** [review]

**Pull request merged**